extensions/AzureCosmosDBMongoVCore/AzureCosmosDBMongoVCore/AzureCosmosDBMongoVCoreConfig.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.KernelMemory;
+
+public class AzureCosmosDBMongoVCoreConfig
+{
+    /// <summary>
+    /// Get more details about Azure Cosmos Mongo vCore https://learn.microsoft.com/en-us/azure/cosmos-db/mongodb/vcore/vector-search
+    /// </summary>
+
+    /// <summary>
+    /// Connection string required to connect to Azure Cosmos Mongo vCore
```

Comment:
pls add an example in the description, or a link to docs to find out what this looks like

Reply:
Added a reference link.